### PR TITLE
Closes #1097 so that zcash-cli displays license info like zcashd.

### LINKED ISSUE #1097: zcash-cli --version, etc. should print the full copyright notice
```
$ ./src/zcash-cli --version
Zcash RPC client version v0.11.2.z6

$ ./src/zcashd --version
Zcash Daemon version v0.11.2.z6
Copyright (C) 2009-2016 The Bitcoin Core Developers
Copyright (C) 2015-2016 The Zcash Developers

This is experimental software.

Distributed under the MIT software license, see the accompanying file COPYING
or <http://www.opensource.org/licenses/mit-license.php>.

This product includes software developed by the OpenSSL Project for use in the
OpenSSL Toolkit <https://www.openssl.org/> and cryptographic software written
by Eric Young and UPnP software written by Thomas Bernard.
```

### PR DESCRIPTION
**Pull request by bitcartel**
LicenseInfo is refactored from init.cpp to util.cpp so that the
bitcoin-cli makefile target does not need to be modified.

### CODE REVIEW

File: src/bitcoin-cli.cpp [hunk 1]
```diff
@@ -9,6 +9,7 @@
 #include "rpcprotocol.h"
 #include "util.h"
 #include "utilstrencodings.h"
+#include "init.h"
```
Comment (str4d) [inline src/bitcoin-cli.cpp]: Is this an accidental inclusion? After moving the license method to `util.h`, it shouldn't be necessary?

Comment (bitcartel) [inline src/bitcoin-cli.cpp]: Yep, due to testing when refactoring.  Fixed now.

**Comment by bitcartel** [comment]
@str4d See reply to comment.

**Comment by ebfull** [comment]
ACK

@zkbot r+

**Comment by ebfull** [comment]
This appears to have uncovered some kind of unrelated race condition in the RPC tests.

**Comment by bitcartel** [comment]
Looks like the test polled for 300 seconds (maximum timeout) waiting for the z_sendmany to finish.  The  test expected a result of 'success' but for some reason it was still 'executing'.  Maybe something happened to the build server at this time to prevent the joinsplit transaction from being created or run very slow.

**Comment by bitcartel** [comment]
@zkbot r+

**Comment by bitcartel** [comment]
@zkbot retry

**Comment by bitcartel** [comment]
A different test involving joinsplit creation has failed.  The timeout used in wallet.py is 120, so increasing it should help.

**Comment by bitcartel** [comment]
@zkbot r+

**Pull request merged**